File: app/src/main/java/ro/cluj/totemz/model/UserGroup.java
package ro.cluj.totemz.model;

import java.util.List;

/**
 * Created by mihai on 7/2/2017.
 */

public class UserGroup {
    public  String name;
    public  TotemzUser owner;
    public  List<TotemzUser> users;

    public UserGroup(String name, TotemzUser owner, List<TotemzUser> users) {
        this.name = name;
        this.owner = owner;
        this.users = users;
    }
    public UserGroup(){

    }
}

File: app/src/main/java/ro/cluj/totemz/model/UserGroup.kt
package ro.cluj.totemz.model

import android.annotation.SuppressLint
import android.os.Parcelable
import kotlinx.android.parcel.Parcelize

/**
 * Created by mihai on 7/2/2017.
 */
@SuppressLint("ParcelCreator")
@Parcelize
data class UserGroup(val name: String, val owner: User, val users: List<User>) : Parcelable
